Core/MIPS/MIPSAnalyst.h

```diff
@@ -144,25 +144,25 @@ namespace MIPSAnalyst
 		u32 opcodeAddress;
 		MIPSOpcode encodedOpcode;
 
-		// shared between branches and conditional moves
-		bool isConditional;
-		bool conditionMet;
-
 		// branches
 		u32 branchTarget;
+		int branchRegisterNum;
```

Comment:
move this one int down again, I liked having it next to isBranchToRegister. It's aligned with no waste anyway since it's after exactly four bools.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/19047/commits/87bd591cee2a17d537b75330467cc699318fd3b9